source/get-started/install-configfile.rst

```diff
@@ -15,67 +16,61 @@ machines.
 Prerequisites
 *************
 
-Ensure that your target system supports the installation:
+For installation onto bare metal, ensure that your target system 
+supports the installation:
 
 * :ref:`system-requirements`
 * :ref:`compatibility-check`
+* :ref:`bootable-usb` of the Live Server image

```

Comment:
I recommend revising this bullet:
`:ref: bootable-usb` of the  `:ref: bare-metal-install-server` .  

Add a note below to explain  to the user that it's only possible to create yaml files with the live-server version. I just want this to be very explicit.

Reply:
That's not true.  The live-desktop version also saves the YAML automatically after installation is completed.

Reply:
Oh, yeah!. You're right.  We never documented that in the live-desktop installation guide. I think we should add a note there too.